messages/run.js

```diff
@@ -28,20 +23,26 @@ module.exports = {
 		"vceDescription": "throws an error when violations are detected",
 		"vceDescriptionLong": "Throws an error when violations are detected. Exit code is the most severe violation.",
 		'engineDescription': "engine(s) to run",
-		'engineDescriptionLong': "One or more engines to run. Multiple values can be specified as a comma-separated list."
+		'engineDescriptionLong': "One or more engines to run. Multiple values can be specified as a comma-separated list.",
+		'eslintConfigDescription': 'location of eslintrc config to customize eslint engine',
+		'eslintConfigDescriptionLong': 'Location of eslintrc to customize eslint engine',
+		'pmdConfigDescription': 'location of PMD rule reference XML file to customize rule selection',
+		'pmdConfigDescriptionLong': 'Location of PMD rule reference XML file to customize rule selection'
 	},
 	"validations": {
 		"mustTargetSomething": "Please specify a codebase using --target.", // TODO: Once --org is implemented, rewrite this message.
 		"outfileFormatMismatch": "Your chosen format %s does not appear to match your output file type of %s.",
 		"outfileMustBeValid": "--outfile must be a well-formed filepath.",
-		"outfileMustBeSupportedType": "--outfile must be of a supported type. Current options are .xml and .csv."
+		"outfileMustBeSupportedType": "--outfile must be of a supported type. Current options are .xml and .csv.",
+		"tsConfigEslintConfigExclusive": "Please provide tsconfig path within the eslint config file under 'parseOptions.project'."
```

Comment:
Would this be clearer if it started with `You can't specify --tsconfig  if you have specified --eslintconfig.`

Reply:
I started with the `exclusive` property in flag definition but felt that some additional information will be helpful. Let me modify it to first state the exclusivity error followed by the workaround.